src/NHibernate/Multi/CriteriaBatchItem.cs

```diff
@@ -1,17 +1,18 @@
+using System;
 using System.Collections.Generic;
 using NHibernate.Impl;
 using NHibernate.Loader.Criteria;
 using NHibernate.Persister.Entity;
 
-namespace NHibernate
+namespace NHibernate.Multi
```

Comment:
I hadn't remarks this till now, but if it i in a `Multi` folder, the namespace has to match. (Changed for all files.)